### ISSUE TITLE
DeformModel confusion about t

### ISSUE THREAD

**Issue by Erhushenshou** [JiaweiXu8/Grid4D#12]
I see in deform_model.py  here at   

def step(self, xyz, t, fixed_attention=False):
        xyz = xyz * self.scale_xyz[None, ...]
        t = (t * 2 * self.grid4d.bound - self.grid4d.bound) * 0.9
        xyzt = torch.cat([xyz, t], dim=-1)

I am curious why t is propossed like this? Why 0.9? Thank you.

**Comment by JiaweiXu8** [comment]
Thanks for your interests. We scale t with 0.9 to eliminate the ambiguity of grid boundaries (when t == 1.0 or 0.0). I think you can also use other values smaller than but near to 1.0 (e.g. 0.95, 0.9) to achieve the same goal.
